src/main/java/reposense/authorship/FileInfoExtractor.java

```diff
@@ -164,6 +191,7 @@ private static void setLinesToTrack(FileInfo fileInfo, String fileDiffResult) {
      * based on {@code config} and inserts it into {@code fileInfos}.
      */
     private static void getAllFileInfo(RepoConfiguration config, Path directory, List<FileInfo> fileInfos) {
+        Set<Path> nonBinaryFilesList = getListOfNonBinaryFiles(config);
         try (Stream<Path> pathStream = Files.list(directory)) {
             for (Path filePath : pathStream.collect(Collectors.toList())) {
```

Comment:
the new implementation should be able to replace the existing pathStream implementation.